src/status_im2/contexts/chat/composer/reply/view.cljs

```diff
@@ -60,15 +60,23 @@
      :style               style/reply-deleted-message}
     (i18n/label :t/message-deleted)]])
 
+(defn- process-identicon
+  "Makes identicon optional"
+  [{:keys [uri] :as photo-path} disable-identicon?]
+  (if disable-identicon?
+    (assoc photo-path :uri (string/replace uri #"&addRing=1" ""))
```

Comment:
does "&addRing=1" mean no ring ?

Reply:
It means ring, Replacing it with an empty string means we won't get a ring back